CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+1. Fork this repository to your own account
+2. Make your changes and verify that tests pass
+3. Commit your work and push to a new branch on your fork
+4. Submit a pull request
+5. Participate in the code review process by responding to feedback
+
+
+### Mac M1 contributors
+In order to launch the project locally, you need to do these preparations:
+1. In file `save-cloud/build.gradle.kts` change languageVersion of `org.liquibase.gradle.LiquibaseTask` from 11 to 17
```

Comment:
We shuold try and change version and see if build passes. Liquibase might have fixed issue with newer JVMs, because we observed failures with Liquibase 4.7, and now it's already 4.9

Reply:
Could you please try in on Windows? On Mac it works fine.

Reply:
On windows I get
```
The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at liquibase.database.ConnectionServiceFactory.create(ConnectionServiceFactory.java:35)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:214)
	... 31 more
Caused by: liquibase.exception.DatabaseException: Connection could not be created to jdbc:mysql://localhost:3306/save_cloud with driver com.mysql.cj.jdbc.Driver.  Communications link failure
```
with JVM 17

Reply:
I've found [this](https://stackoverflow.com/a/67438592) SO answer, and adding this flag to URL takes things somewhat further, but still ends with:
```
[2022-04-22 14:01:03] SEVERE [liquibase.integration] Cannot invoke "java.io.InputStream.available()" because "is" is null
liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: Unexpected error running Liquibase: Cannot invoke "java.io.InputStream.available()" because "is" is null
```

Reply:
oh ok thanks

Reply:
Made it to choose java version according to os family.

Reply:
Funny thing, it actually works on Windows, if you *launch* gradle itself with JVM 16 or 17, i.e. `JAVA_HOME` points to a new Java. With `JAVA_HOME` being Java 8 or 11 or even 13 `liquibaseUpdate` fails with Java 17 in toolchains. Very strange behavior which actually looks like bug on the side of Gradle.